### PR TITLE
shoe-312: added background color body

### PR DESCRIPTION
**Pull request by developerjaym**
I added a background color to the body! It'll be red now

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: red;
```
Comment (mlgvla) [inline index.html]: This really looks pretty bad.  Maybe something lighter?

Comment (developerjaym) [reply index.html]: Whatever

Comment (developerjaym) [reply index.html]: I've updated it to a light red. Does this please you NOW?

Comment (mlgvla) [reply index.html]: Way better!!!  Love it!

File: index.html [hunk 2]
```diff
@@ -5,6 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: rgb(186, 138, 138);
```
Comment (mlgvla) [inline index.html:10]: Way better!

**Review by mlgvla — APPROVED** [review]

**Pull request closed without merging**